index.html

```diff
@@ -0,0 +1,137 @@
+          <div class="service-card">
+            <div class="service-image">
+              <img
+                src="https://th.bing.com/th/id/OIP.ofZc6YfimQ0mpHZIShtgsAHaF-?w=259&h=209&c=7&r=0&o=5&dpr=1.3&pid=1.7"
+                alt=""
+              />
+            </div>
+            <div class="service-text">
+              <h2>Premium Materials</h2>
+              <p>
+                Our trombones use the shiniest brass which is sourced locally.
+                This will increase the longevity of your purchase.
+              </p>
+            </div>
+          </div>
+          <div class="service-card">
+            <div class="service-image">
+              <img
+                src="https://th.bing.com/th/id/OIP.ofZc6YfimQ0mpHZIShtgsAHaF-?w=259&h=209&c=7&r=0&o=5&dpr=1.3&pid=1.7"
+                alt=""
+              />
+            </div>
+            <div class="service-text">
+              <h2>Fast Shipping</h2>
+              <p>
+                Our trombones use the shiniest brass which is sourced locally.
+                This will increase the longevity of your purchase.
+              </p>
+            </div>
+          </div>
+          <div class="service-card">
+            <div class="service-image">
+              <img
+                src="https://th.bing.com/th/id/OIP.ofZc6YfimQ0mpHZIShtgsAHaF-?w=259&h=209&c=7&r=0&o=5&dpr=1.3&pid=1.7"
+                alt=""
+              />
+            </div>
+            <div class="service-text">
+              <h2>Quality Assurance</h2>
+              <p>
+                Our trombones use the shiniest brass which is sourced locally.
+                This will increase the longevity of your purchase.
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="youtube-section">
+        <a href="https://youtu.be/y8Yv4pnO7qc">
+          <img
+            src="https://th.bing.com/th/id/OIP.bXrHc29g4Cy3E5QH5imMjQHaFf?w=223&h=180&c=7&r=0&o=5&dpr=1.3&pid=1.7"
+            alt=""
+          />
+        </a>
+      </div>
+      <div class="pricing-options">
+        <div class="price-card">
+          <h2>Tenor Trombone</h2>
+          <h2 class="pricing-price">$600</h2>
+          <p class="pricing-text">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptate,
+            dolor explicabo. Reiciendis impedit maiores voluptas nesciunt illum
+            officia doloribus aperiam.
+          </p>
+        </div>
+        <div class="price-card">
+          <h2>Tenor Trombone</h2>
+          <h2 class="pricing-price">$600</h2>
+          <p class="pricing-text">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptate,
+            dolor explicabo. Reiciendis impedit maiores voluptas nesciunt illum
+            officia doloribus aperiam.
+          </p>
+        </div>
+        <div class="price-card">
+          <h2>Tenor Trombone</h2>
+          <h2 class="pricing-price">$600</h2>
```

Comment:
I would suggest you use an h3 tag here as an h2 tag is already for better hierarchy and professionalism

Reply:
Hi, @Samuelwanza!! 👋
- Here you have made a good suggestion with reasoning. 👍 

- [ ] Kindly phrase the comments politely as suggestions instead of orders.
  - For example:
    - This is a polite suggestion: "I think it would be better if you use an h3 tag.......". 
- [ ] Kindly use emoticon (😄) so that your review looks human-friendly.